ydb/core/blobstorage/vdisk/balance/balancing_actor.cpp

```diff
@@ -160,6 +164,11 @@ namespace NBalancing {
                 }
 
                 merger.Clear();
+
+                if (SendOnMainParts.Size() >= MAX_TO_SEND_PER_EPOCH && TryDeleteParts.Size() >= MAX_TO_DELETE_PER_EPOCH) {
```

Comment:
maybe || ?

if any of limits is exceeded actor should stop.

Reply:
No, thats not a mistake, I check `TryDeleteParts.Size() < MAX_TO_DELETE_PER_EPOCH` and `SendOnMainParts.Size() < MAX_TO_SEND_PER_EPOCH` above before pushing to queues. If I will check `||`, there could be a situation when I will only send parts on main for a long time, without deletes